package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@equinor/fusion-components",
-    "version": "1.3.13",
+    "version": "1.3.13-pr792.14",
```

Comment:
This is for testing right? Should be removed before merge

Reply:
this will change before when versjoning